src/main/java/rosenfeld/boggle/Game.java

```diff
@@ -0,0 +1,63 @@
+package rosenfeld.boggle;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    Dictionary dictionary = new Dictionary();
+    private List<String> userWords = new ArrayList<>();
+    private final int MIN_LENGTH = 4;
+
+    public Game() throws IOException {
```

Comment:
Don't declare that you are throwing exceptions that you aren't actually throwing.

Reply:
Because I'm creating an instance of Dictionary, it gave me an error unless I added the throws line.

Reply:
Pass the Dictionary into Game's constructor instead